onnxruntime/test/python/onnxruntime_test_python_iobinding.py

```diff
@@ -10,7 +10,7 @@
 import onnxruntime as onnxrt
 from onnxruntime.capi._pybind_state import OrtDevice as C_OrtDevice  # pylint: disable=E0611
 from onnxruntime.capi._pybind_state import OrtValue as C_OrtValue
-from onnxruntime.capi._pybind_state import SessionIOBinding
+from onnxruntime.capi._pybind_state import OrtValueVector, SessionIOBinding
```

Comment:
Minimal build probably wont have access to dlpack.  suggest double check on that as I am not 100% sure

Reply:
I was able to build minimal build so I asume it has access.

Reply:
Good to know. Does minimal build can actually leverage this code? I know they dont use ORTModule or any training code. Maybe InferenceSession is used ad DLPack is useful for them. If not, better not to compile it to keep binary size minimal

Reply:
Sorry, I did this a long time ago. I checked again and dlpack is only enabled when ENABLE_TRAINING is true. I assumed this was only needed when exchanging vectors with pytorch so I let it that way. Minimal build probably don't use dlpack, not for this at least.